release/README-maintainers.html

```diff
@@ -225,6 +236,15 @@
         </tr>
         <tr>
             <td>8.</td>
+            <td><strong>Ensure Travis displays the status of typetools projects</strong><br/>
+                Sign in to <a href="https://travis-ci.org/">Travis</a> and ensure that the left-hand column displays status
+                for typetools projects (scrolling is required).
+                <!-- What should be done if the projects are not displayed? This just worked for me so I am not sure what
```

Comment:
It will work if you are a member of the GitHub typetools repository.  If that's not true then many steps in the release process won't work.  I think you can delete this comment.